pkg/macvlan/macvlan.go

```diff
@@ -403,16 +413,24 @@ func macvlanCmdAdd(args *skel.CmdArgs) error {
 		}
 
 		// Add route to gateway on macvlan interface
-		destIpNet := net.IPNet{
-			IP:   gw,
-			Mask: net.CIDRMask(32, 32),
+		var destIpNet net.IPNet
+		if is_v6 {
+			destIpNet = net.IPNet{
+				IP:   gw,
+				Mask: net.CIDRMask(64, 64),
```

Comment:
Why is the mask size 64 and not 128?

Reply:
I am not very familiar with this functionality, so it's a question not a complaint.

Reply:
actually your right need to be full range like v4